app_dart/test/src/request_handling/fake_http.dart

```diff
@@ -563,7 +563,7 @@ class FakeHttpHeaders implements HttpHeaders {
   List<String> operator [](String name) => _values[name];
 
   @override
-  void add(String name, Object value) {
+  void add(String name, Object value, {bool preserveHeaderCase = false}) {
```

Comment:
Was there an update to the http library?

Reply:
Yes.